Automattic/wp-calypso
# Performance Profiler: Add topbar information

Fixes [Performance Profiler: Create top bar area for result page](https://github.com/Automattic/dotcom-forge/issues/8564)

## Proposed Changes

Add TopBar information as:
* Tested date (already implemented)
* Hosted on WP.com information
* Share option (functionality to be implemented on [Performance Profiler: Add ability to share the results](https://github.com/Automattic/dotcom-forge/issues/8822))
![CleanShot 2024-08-20 at 17 47 45@2x](https://github.com/user-attachments/assets/2e494fd8-7981-4620-80ed-5c560acefc45)

## Why are these changes being made?

[Performance Profiler: Create top bar area for result page](https://github.com/Automattic/dotcom-forge/issues/8564)

## Testing Instructions

 * Go to `/speed-test-tool?url=wordpress.com`
 * Check if the all the information is displayed as above
 * To force the WP.com badge to be shown, on your dev environment replace the piece `showWPcomBadge={ performanceReport?.is_wpcom }` by `showWPcomBadge`

## Pre-merge Checklist
